### PR TITLE
Two new Magnum guns, few new caps, new hunter sprites, radio-backpack

### PR DESCRIPTION
**Pull request by SouthyOsmanli**
## About The Pull Request
<details>
<summary>
	Sort of misc changes I'm putting up before any major changes. Two new magnum caliber guns have been added; the survival rifle 'Bond' - a magnum caliber carbine, and a magnum revolver with awesome sprites from Mojave sun; sprited by **Inferred Baron**. I changed the colors a bit to fit Eris but the sprite itself is his work; not mine - check out Mojave Sun for his awesome sprites.
	
Added three new hats; two being for a costume and the other a Blackshield radio hat to reference a long-range radio piece. Currently a WIP. New radio backpack to fit the radio hat sprite, currently has no new use but plans to use both together in the future at some point. Added new hunter jacket to its loadout for the lodge leader, mild changes on the other hunter armor pieces to make them look a bit better, redid the hunting lodge helmet which may be redone once more at a later point but at least it fits the rest of the style now.
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
add: Radio hat (admin spawn only)
add: Anarchist hat & cap (admin spawn only)
add: Hunting lodge jacket (unused currently but planned near-future usage)
add: New radio backpack sprite
add: 'Bond' Survival Rifle, as asked for by Trilby
add: 10mm styled magnum revolver from Mojave Sun; all credit to Inferred Baron
add: Two new Excelsior posters; _Stalin approves_
/:cl:

### CODE REVIEW

**Comment by SouthyOsmanli** [comment]
Reason for pushing this so unorganized like is accidently doing my gun changes on my own 'for fun' uniform project. I removed more iffy stuff, such as the uniforms I had made originally. Plan is to try an ammo overhaul for .35 rounds after this PR thus why I've put it up in its current state.

File: code/modules/projectiles/guns/projectile/revolver/rev10.dm [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	desc = "A well made police revolver used by many departments round the universal frontier. While the design appears to predate its production by H&S it has become iconic to their company for stopping power. \
+	It appears to take .40 magnum rounds."
+	icon = 'icons/obj/guns/projectile/rev10.dmi'
+	icon_state = "rev10"
+	item_state = "rev10"
+	drawChargeMeter = FALSE
+	caliber = CAL_MAGNUM
+	origin_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 2)
+	max_shells = 6
+	ammo_type = /obj/item/ammo_casing/c10x24
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/revolver/rev10.dm:12]:
```suggestion
```

File: code/modules/projectiles/guns/projectile/automatic/survival_rifle.dm [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/obj/item/weapon/gun/projectile/automatic/survivalrifle
+	name = "\"Bond\" survival rifle"
+	desc = "A \"Bond\" marksman carbine manufactured by Hunters Inc. Compact, high velocity and high penetration despite its low fire rate. Uses the obscure 12x24 rounds."
+	icon = 'icons/obj/guns/projectile/survivalrifle.dmi'
+	icon_state = "survivalrifle"
+	item_state = "survivalrifle"
+	w_class = ITEM_SIZE_BULKY
+	force = WEAPON_FORCE_PAINFUL
+	mag_well = MAG_WELL_PISTOL
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/automatic/survival_rifle.dm:9]:
```suggestion
```

File: code/modules/projectiles/guns/projectile/automatic/survival_rifle.dm [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	caliber = CAL_MAGNUM
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/automatic/survival_rifle.dm:10]:
```suggestion
	ammo_type =  /obj/item/ammo_casing/c10x24

```

File: code/modules/projectiles/guns/projectile/automatic/survival_rifle.dm [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	origin_tech = list(TECH_COMBAT = 6, TECH_MATERIAL = 1)
+	slot_flags = SLOT_BACK
+	load_method = MAGAZINE
+	mag_well = MAG_WELL_PULSE
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/automatic/survival_rifle.dm:14]:
```suggestion
	mag_well = MAG_WELL_RIFLE
```

File: code/modules/projectiles/guns/projectile/automatic/survival_rifle.dm (hunk 3 above)
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/automatic/survival_rifle.dm:10]:
```suggestion
	caliber = "c10x24"
```

File: code/modules/projectiles/guns/projectile/automatic/survival_rifle.dm [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+	unload_sound 	= 'sound/weapons/guns/interact/ltrifle_magout.ogg'
+	reload_sound 	= 'sound/weapons/guns/interact/m41_reload.ogg'
+	cocked_sound 	= 'sound/weapons/guns/interact/m41_cocked.ogg'
+	zoom_factor = 0.6
+	recoil_buildup = 12
+	one_hand_penalty = 20 //automatic rifle level
+
+	init_firemodes = list(
+		SEMI_AUTO_NODELAY)
+
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/automatic/survival_rifle.dm:29]:
```suggestion
/obj/item/ammo_magazine/rifle_10x24
	name = "light rifle magazine"
	desc = "A 20 round magazine marked for 12x24 rounds."
	icon_state = "rifle"
	mag_type = MAGAZINE
	mag_well = MAG_WELL_RIFLE
	matter = list(MATERIAL_STEEL = 8)
	caliber = "10x24"
	ammo_type = /obj/item/ammo_casing/c10x24
	max_ammo = 20
	multiple_sprites = 1
```
something like this or something

**Review by Trilbyspaceclone — APPROVED** [review]
:Stalinapproves:

File: code/modules/projectiles/guns/projectile/revolver/rev10.dm [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+	gun_tags = list(GUN_PROJECTILE, GUN_INTERNAL_MAG, GUN_REVOLVER)
+
+
+/obj/item/weapon/gun/projectile/rev10/update_icon()
+	..()
+
+	var/iconstring = initial(icon_state)
+	var/itemstring = ""
+
+	if (!ammo_magazine || !length(ammo_magazine.stored_ammo))
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/projectile/revolver/rev10.dm:28]:
```suggestion
	if(len==0)
```
try this

**Pull request merged**